pyproject.toml

```diff
@@ -86,7 +86,7 @@ include-package-data = true
 #packages = ["src/omotes_simulator_core"]
 
 [tool.pytest.ini_options]
-# addopts = "--cov=omotes_simulator_core --cov-report html --cov-report term-missing --cov-fail-under 80"
+addopts = "--cov=omotes_simulator_core --cov-report html --cov-report term-missing --cov-fail-under 80"
```

Comment:
Why do you need to change this?

Reply:
To get the code coverage working again on github action. This line should not be turned off